.github/workflows/ci.yaml

```diff
@@ -13,17 +13,23 @@ jobs:
 
       # make sure all code has been formatted with rustfmt
       - run: rustup component add rustfmt
-      - uses: actions-rs/cargo@v1
+      - name: check rustfmt
+        uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --all -- --check --color always
+          args: -- --check --color always
 
       # run clippy to verify we have no warnings
       - run: rustup component add clippy
-      - uses: actions-rs/cargo@v1
+      - name: cargo fetch
+        uses: actions-rs/cargo@v1
+        with:
+          command: fetch
+      - name: cargo clippy
+        uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-features -- -D warnings
+          args: --lib --tests -- -D warnings
```

Comment:
`--all-features` would still be good to have here right? though this specific project may not have optional features that are disabled by default

Reply:
Yes, the only feature we have is `progress` which only applies to the CLI.